tests/test.js

```diff
@@ -1,131 +1,178 @@
+  it("should return the same number of words if the same seed is used", function () {
+    const seed = "seed1";
+    const min = 1;
+    const max = 10;
 
-        const words = randomWords({ seed, min, max });
-        const words2 = randomWords({ seed, min, max });
+    const words = randomWords({ seed, min, max });
+    const words2 = randomWords({ seed, min, max });
 
-        assert.ok(words.length == words2.length, 'number of words is the same')
-    });
-    it('should return different words if no seeds are provided', function () {
-        const exactly = 20;
-        const join = ' ';
+    assert.ok(words.length == words2.length, "number of words is the same");
+  });
+  it("should return different words if no seeds are provided", function () {
+    const exactly = 20;
+    const join = " ";
 
-        const words = randomWords({ exactly, join });
-        const words2 = randomWords({ exactly, join });
+    const words = randomWords({ exactly, join });
+    const words2 = randomWords({ exactly, join });
 
-        // with 1952 possible words, at least one word in 20 should be different
-        assert.ok(words != words2, 'words are different')
-    });
-    it('should return different words if different seeds are used', function () {
-        const exactly = 20;
+    // with 1952 possible words, at least one word in 20 should be different
+    assert.ok(words != words2, "words are different");
+  });
+  it("should return different words if different seeds are used", function () {
+    const exactly = 20;
 
-        const words = randomWords({ seed: 'seed1', exactly });
-        const words2 = randomWords({ seed: 'seed2', exactly });
+    const words = randomWords({ seed: "seed1", exactly });
+    const words2 = randomWords({ seed: "seed2", exactly });
 
-        // with these seeds, all words should be different
-        for (let i = 0; i < exactly; i++) {
-            assert.ok(words[i] != words2[i], 'words are different')
-        }
-    });
-    it('should return different number of words if different seeds are used', function () {
-        const min = 1;
-        const max = 10;
+    // with these seeds, all words should be different
+    for (let i = 0; i < exactly; i++) {
+      assert.ok(words[i] != words2[i], "words are different");
+    }
+  });
+  it("should return different number of words if different seeds are used", function () {
+    const min = 1;
+    const max = 10;
 
-        const words = randomWords({ seed: 'seed1', min, max });
-        const words2 = randomWords({ seed: 'seed2', min, max });
+    const words = randomWords({ seed: "seed1", min, max });
+    const words2 = randomWords({ seed: "seed2", min, max });
 
-        // with these seeds, the number of words should 5 and 3
-        assert.ok(words.length != words2.length, 'number of words is different')
-    });
+    // with these seeds, the number of words should 5 and 3
+    assert.ok(words.length != words2.length, "number of words is different");
+  });
```

Comment:
Can you please write 4 additional tests. One is to test that passing just `minLength` works, one showing that `maxLength` alone works. This tests your formally redundant code blocks. One showing that passing a `maxLength` larger than the biggest word does not result in an endless loop, and finally, one that shows passing a `minLength` larger than the longest word doesn't result in an endless loop.